core/domain/exp_domain.py

```diff
@@ -2734,6 +2734,58 @@ def _convert_states_v34_dict_to_v35_dict(cls, states_dict):
 
         return states_dict
 
+    @classmethod
+    def _convert_states_v35_dict_to_v36_dict(cls, states_dict):
+        """Converts from version 35 to 36. Version 36 adds a customization arg
+        for the Math interactions that allows creators to specify the letters
+        that would be displayed to the learner.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        greek_symbols = constants.GREEK_SYMBOLS_LOWERCASE + (
+            constants.GREEK_SYMBOLS_UPPERCASE)
+        greek_letters = constants.GREEK_LETTERS
```

Comment:
What's the difference? Is one the display value and one the input value?

Reply:
`GREEK_LETTERS` represent the words and `GREEK_SYMBOLS` represent the symbols. I thought naming it to something like `GREEK_WORDS` seems weird which is why I chose to name it this way.

Reply:
I suggest going with GREEK_LETTER_NAMES -- that would be less confusing, IMO.